.eslintrc.js

```diff
@@ -0,0 +1,26 @@
+module.exports = {
+  env: {
+    browser: true,
+    es6: true,
+    node: true,
+  },
+  extends: [
+    'airbnb',
+  ],
+  globals: {
+    Atomics: 'readonly',
+    SharedArrayBuffer: 'readonly',
+  },
+  parserOptions: {
+    ecmaFeatures: {
+      jsx: true,
+    },
+    ecmaVersion: 2018,
+    sourceType: 'module',
+  },
```

Comment:
I was going to ask where you got these -- I don't have them. Then I found them in a file made by some eslint init command. I didn't use any of them though. I just use below. But I think this is a good test case -- maybe mine will break and I'll know to try yours (and vice-versa.)

  "extends": ["airbnb"],
  "env": {
    "jest": true,
    "browser": true,

Reply:
Yes, I run `npx eslint --init` (reference: https://eslint.org/docs/user-guide/getting-started). That's a good point. Maybe I don't need those.